packages/shared-ux/page/analytics_no_data/impl/src/analytics_no_data_page.tsx

```diff
@@ -20,14 +21,17 @@ export const AnalyticsNoDataPage = ({
   allowAdHocDataView,
 }: AnalyticsNoDataPageProps) => {
   const services = useServices();
-  const { kibanaGuideDocLink } = services;
+  const { kibanaGuideDocLink, customBranding } = services;
+
+  const { hasCustomBranding$ } = customBranding;
 
   return (
     <Component
       {...{
         onDataViewCreated,
         allowAdHocDataView,
         kibanaGuideDocLink,
+        showPlainSpinner: useObservable(hasCustomBranding$) ?? false,
```

Comment:
Let's make this a constant on line 26: `const hasCustomBranding = useObservable(customBranding.hasCustomBranding$`